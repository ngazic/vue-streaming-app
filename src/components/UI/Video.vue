<template>
<div class="container">
  <div class="row justify-content-md-center">
    <div class="col col-sm-10 ">
      <h1>{{ getCurrentVideo().title }}</h1>
      <video vjs-big-play-centered :src="getCurrentVideo().url" muted controls class="video-js" data-setup="{}"></video>
    </div>
  </div>
</div>
</template>

<script lang="ts">
import {
  mapGetters
} from 'vuex'
export default {
  methods: {
    ...mapGetters(["getCurrentVideo"])
  }
};
</script>

<style lang="scss" scoped>
.video-js {
  width: 100%;
}
</style>
