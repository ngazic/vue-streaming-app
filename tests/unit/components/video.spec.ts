import { shallowMount, createLocalVue } from '@vue/test-utils'
import VueRouter from 'vue-router'
import Vuex from 'vuex'
import Video from '@/components/UI/Video.vue'

describe('Video.vue', () => {

  it('renders correct video from Vuex', () => {
    expect(true).toBe(false)
  });
})