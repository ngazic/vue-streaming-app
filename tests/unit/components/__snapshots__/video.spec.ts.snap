// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video.vue show/hide links on authentification renders correctly 1`] = `
<div class="container">
  <div class="row justify-content-md-center">
    <div class="col col-sm-10 ">
      <h1>video 2</h1> <video vjs-big-play-centered="" src="https://ak.picdn.net/shutterstock/videos/1054068659/preview/stock-footage-group-of-young-boys-play-soccer-training-day-on-the-football-field-teenagers-play-football-on-a.webm" muted="muted" controls="controls" data-setup="{}" class="video-js"></video>
    </div>
  </div>
</div>
`;
